hello_wifi/mix.exs

```diff
@@ -75,7 +70,7 @@ defmodule HelloWifi.Mixfile do
      "deps.loadpaths":  ["deps.loadpaths", "nerves.loadpaths"]]
   end
 
-  def kernel_modules("rpi0"), do: ["brcmfmac"]
+  def kernel_modules("rpi0"), do: []
```

Comment:
This can be changed back to "brcmfmac" now that the rpi0 system has the changes you propoosed to make brcmfmac a module for the sake of consistency across systems.